modules/ROOT/pages/amazon/amazon-ec2-connector.adoc

```diff
@@ -516,27 +519,34 @@ HTTP Listener component.
 |===
 |Studio Field |Value
 |Display Name |Listener
-|Extension Configuration | If no HTTP Listener configuration exists, click the plus sign (+) to add a new HTTP Listener configuration, providing the values `localhost` and `8081` for the Host and Port, and click *OK*.
+|Extension Configuration a| If an HTTP Listener configuration doesn't exist: 
+
+1. Click the plus sign (+) to add a new HTTP Listener configuration
+2. Provide the following values: +
+* Host: `localhost`
+* Port: `8081`
+3. Click *OK*.
 |Path |/createVolume
 |===
 +
 . Drag the Amazon EC2 Connector *Create volume* operation next to the HTTP Listener component.
 . Configure the EC2 connector by adding a new Amazon EC2 global element and click *+* next to the *Connector Configuration* field.
-.. Configure the global element according as follows:
+. Configure the global element as follows:
 +
 
 [%header,cols="30s,70a"]
 |===
-|Studio Field |Description |Value
-|Name |Enter a name for other instances of the connector to reference it |`<Configuration_Name>`
-|Access Key |Alphanumeric text string that uniquely identifies the user who owns the account |`${amazon.accesskey}`
-|Secret Key |Key that plays acts as a password |`${amazon.secretkey}`
-|Region Endpoint |Region to select from the drop-down menu for the Amazon EC2 client |`USEAST1`
+|Studio Field |Description
+|Name |Enter a name for other instances of the connector to reference it
```

Comment:
This is worded as a sentence, but the other descriptions aren't. I would change this to "Name by which to reference the connector," or something like that.